hpke/vectors_test.go

```diff
@@ -228,16 +228,17 @@ type vector struct {
 	KdfID              uint16             `json:"kdf_id"`
 	AeadID             uint16             `json:"aead_id"`
 	Info               string             `json:"info"`
+	Iem                string             `json:"iem,omitempty"`
```

Comment:
Is this "input encapsulation material"? Can we maybe rename it to "input encapsulation randomness" or "input encapsulation seed"?

Reply:
Renamed to  "input encapsulation randomness"